bin/i18n/resources/dashboard/course_content/sync_in.rb

```diff
@@ -200,6 +200,26 @@ def process
                   i18n_strings[BLOCK_CATEGORIES_TYPE][name] = name if name
                 end
 
+                ## Lab2 Function Names
```

Comment:
nit: maybe add that Lab2 Blockly labs use JSON (as opposed to XML used elsewhere here)?

Reply:
Good call.